.gitignore

```diff
@@ -97,4 +97,3 @@ Thumbs.db
 # Ignore built ts files
 __tests__/runner/*
 lib/
-dist/
```

Comment:
This is normal for actions, right?

Reply:
It varies. Some include the built action on the main branch, some don't. I prefer not committing these build outputs but then we'd have to build a release branch for each release. For a repo with just one action that's not a big issue, but here we'd end up with one release branch per action per version which seems like a nuisance to manage.

If we commit it, we can simply tag the commit on the main branch as version.

Reply:
Actually, nvm that. I already have a release script for building release branches and it would only require minor adjustments to support several actions in one repo: https://github.com/julia-actions/bin/blob/9ceca17c9cc3b29385880fcf449dbfe5e4443a98/build-release.sh

That way we won't have to deal with rebuilding dev branches locally at all.

Reply:
That also solves the "stay in sync issue" which I hate. So yeah if we can build programmatically instead, that'd be perfect.

Reply:
0856d11 fixes this